README.rst

```diff
@@ -1,10 +1,31 @@
 .. image:: https://circleci.com/gh/dwavesystems/structural-imbalance-demo.svg?style=svg
     :target: https://circleci.com/gh/dwavesystems/structural-imbalance-demo
 
-Demo for the Structural Imbalance Project
-=========================================
+Demo of Structural Imbalance in Signed Social Networks
+======================================================
 
-Data is from the Stanford Militants Mapping Project
+*Social networks* map relationships between people or organizations onto graphs, with
+the people/organizations as nodes and relationships as edges; for example,
+Facebook friends form a social network with friends represented as
+nodes with connecting edges. *Signed social networks* map both friendly and
+hostile relationships by setting the values of the edges between nodes with either plus ("+")
+or minus ("-") signs. Such networks are said to be *structurally balanced* when they
+can be clearly divided into two subgraphs with positive-valued edges (friendly
```

Comment:
These aren't technically subgraphs, since we also care about the edges between the two groups. (The subgraphs induced by these vertices would only include the friendly relationships within each group.) Instead, "their vertices can be clearly divided into two disjoint sets," or something to that effect. This will also affect the two subsequent uses of this term in this sentence.

Reply:
Fixed